src/components/ModelTableList/StatusGroup.test.js

```diff
@@ -59,4 +59,20 @@ describe("StatusGroup", () => {
     );
     expect(wrapper.find("tbody TableRow").length).toBe(3);
   });
+
+  it("displays the provider type icon", () => {
```

Comment:
Should this be expanded to test for each icon type?

You can clone the `dataDump` to hack in different provider names if you need.

Reply:
Happy to but not sure it really gives us that much. At that point, it's a switch case. In the future, I would probably make this its own component so it can be clearing unit tested.